fix: 게시글 삭제 오류

<!-- Issue 작성 전에 우선 Assignees, label 지정하기 -->
## 📄 이슈 내용
<!-- 이슈 내용 요약 설명 -->
> 외래키 문제로 인해 삭제가 되지 않습니다. 좋아요 및 댓글이 함께 삭제되도록 수정합니다.
Caused by: java.sql.SQLIntegrityConstraintViolationException: Cannot delete or update a parent row: a foreign key constraint fails (`aliendb`.`community_article_like_entity`, CONSTRAINT `FKch6jqk7h5b01154xd6huds48b` FOREIGN KEY (`community_article_id`) REFERENCES `community_article_entity` (`id`))

## ✅ 체크리스트
- [x] 게시글 삭제 시 연관된 레코드 함께 삭제하도록 수정

## ➕ETC

--- a/db/src/main/java/com/aliens/db/communityarticle/entity/CommunityArticleEntity.java
+++ b/db/src/main/java/com/aliens/db/communityarticle/entity/CommunityArticleEntity.java
@@ -2,12 +2,16 @@ package com.aliens.db.communityarticle.entity;
 
 import com.aliens.db.BaseEntity;
 import com.aliens.db.communityarticle.ArticleCategory;
+import com.aliens.db.communityarticlecomment.entity.CommunityArticleCommentEntity;
+import com.aliens.db.communityarticlelike.entity.CommunityArticleLikeEntity;
 import com.aliens.db.member.entity.MemberEntity;
 import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
 
 @Getter
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
@@ -27,6 +31,12 @@ public class CommunityArticleEntity
 
     @ManyToOne(fetch = FetchType.LAZY)
     private MemberEntity member;
+
+    @OneToMany(mappedBy = "communityArticle", orphanRemoval = true)
+    private List<CommunityArticleCommentEntity> comments = new ArrayList<>();
+
+    @OneToMany(mappedBy = "communityArticle", orphanRemoval = true)
+    private List<CommunityArticleLikeEntity> likes = new ArrayList<>();
 
     private CommunityArticleEntity(
             String title,

--- a/db/src/main/java/com/aliens/db/marketarticle/entity/MarketArticleEntity.java
+++ b/db/src/main/java/com/aliens/db/marketarticle/entity/MarketArticleEntity.java
@@ -3,12 +3,16 @@ package com.aliens.db.marketarticle.entity;
 import com.aliens.db.BaseEntity;
 import com.aliens.db.marketarticle.MarketArticleStatus;
 import com.aliens.db.marketarticle.ProductStatus;
+import com.aliens.db.marketarticlecomment.entity.MarketArticleCommentEntity;
+import com.aliens.db.marketbookmark.entity.MarketBookmarkEntity;
 import com.aliens.db.member.entity.MemberEntity;
 import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
 
 @Getter
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
@@ -35,6 +39,12 @@ public class MarketArticleEntity
 
     @ManyToOne(fetch = FetchType.LAZY)
     private MemberEntity member;
+
+    @OneToMany(mappedBy = "marketArticle", orphanRemoval = true)
+    private List<MarketArticleCommentEntity> comments = new ArrayList<>();
+
+    @OneToMany(mappedBy = "marketArticle", orphanRemoval = true)
+    private List<MarketBookmarkEntity> likes = new ArrayList<>();
 
     private MarketArticleEntity(
             String title,

